src/react-apps/applications/service-development/src/features/handleMergeConflict/HandleMergeConflictContainer.tsx

```diff
@@ -49,7 +49,6 @@ const styles = () => createStyles({
     boxShadow: theme.sharedStyles.boxShadow,
   },
   title: {
```

Comment:
this is now empty and should be removed, all references should also be removed.

Reply:
Removed unused styling and reference.

Reply:
Resolved